compiler/rustc_ast_passes/src/ast_validation.rs

```diff
@@ -877,6 +878,26 @@ impl<'a> AstValidator<'a> {
             _ => {}
         }
     }
+
+    fn maybe_lint_missing_abi(&mut self, span: Span, id: NodeId) {
+        // FIXME(davidtwco): This is a hack to detect macros which produce spans of the
+        // call site which do not have a macro backtrace. See #61963.
```

Comment:
Should we have some way to enable this to push the proc-macro ecosystem in the right direction?